README.md

```diff
@@ -9,3 +9,23 @@ might help with the notation for mathematical expressions.
 $G_1=(V_1 , E_1)$ is isomorphic to $G_2 = (V_2, E_2)$ if there exists a
 one-to-one and onto function (bijection) $f: V_1 \rightarrow V_2$ such that $(u,v)
 \in E_1$ iff $(f(u),f(v)) \in E_2$.
+
+## Answer 
+Direct Proof:
+- If we have two graphs $A$ and $B$ that are both completely connected and have the same number of nodes.
```

Comment:
You cannot show this using an example.

Reply:
I tried writing this using a more general form. If it needs corrections please let me know.